index.js

```diff
@@ -111,18 +113,22 @@ export class Placement {
    * string from API response. Can also be null, indicating a noop action.
    */
   fetch() {
-    const id = "ad_" + Date.now();
+    const callback = "ad_" + Date.now();
+    var div_id = callback;
+    if (typeof this.target.id !== 'undefined') {
```

Comment:
This line does not appear to be correct. I tested this on a div without an ID and this IF block is still entered. Since `this.target` is always defined and has a value, you can probably do:

```suggestion
    if (this.target.id) {
```